migration/v2.18.0-ck8s1-v2.19.x-ck8s1/migrate-to-containerd.yml

```diff
@@ -129,6 +128,14 @@
           retries: 30
           delay: 3
 
+        - name: Delete remaining docker files
+          file:
+            state: absent
+            path: "{{ item }}"
+          with_items:
+            - '/var/lib/docker/image'
+            - '/var/lib/docker/volumes'
```

Comment:
Are we sure that these are the once consuming space?

Reply:
To me it looks like the overlay2 directory is the one consuming space
```
ubuntu@lingio-production-wc-k8s-node-worker-1:~$ sudo du -csh /var/lib/docker/volumes
52K	/var/lib/docker/volumes
52K	total
ubuntu@lingio-production-wc-k8s-node-worker-1:~$ sudo du -csh /var/lib/docker/image
80M	/var/lib/docker/image
80M	total
ubuntu@lingio-production-wc-k8s-node-worker-1:~$ sudo du -csh /var/lib/docker/overlay2
34G	/var/lib/docker/overlay2
34G	total
```

Reply:
Hmm, yes it sure looks like it. I assume that we only really need to keep `/var/lib/docker/containers` as the logs are in there.